web/src/compose_reply.js

```diff
@@ -149,7 +153,10 @@ export function quote_and_reply(opts) {
 
     compose_ui.insert_syntax_and_focus(quoting_placeholder, $textarea, "block");
 
-    function replace_content(message) {
+    function replace_content(
+        message,
+        raw_content = opts.selected_message_content_raw ?? message.raw_content,
```

Comment:
Could this line be outside the function definition? It feels like we should either be passing a required `raw_content` parameter that the callers pass as appropriate, or just computing this on the first line; I tend to prefer the "pass a real parameter" approach.